beekeeper-cleanup/src/main/java/com/expediagroup/beekeeper/cleanup/hive/HiveClient.java

```diff
@@ -105,6 +106,18 @@ public boolean tableExists(String databaseName, String tableName) {
     }
   }
 
+  @Override
+  public boolean tableHasProperty(String databaseName, String tableName, String property, String value) {
```

Comment:
maybe this can be Map<> getTableProperties() so you need just once call to check multiple properties. Will be a bit more future proof

Reply:
ok, i was considering that